test/unittests/CMakeLists.txt

```diff
@@ -167,19 +174,109 @@ set (CVMFS_UNITTEST_SOURCES
   ${CVMFS_SOURCE_DIR}/uid_map.h
 )
 
-set (CVMFS_UNITTEST_DEBUG_SOURCES ${CVMFS_UNITTEST_SOURCES})
+set (CVMFS_UNITTEST_DEBUG_SOURCES 
+
+  # unit test files
+  ${CVMFS_UNITTEST_FILES}
+  
+  # cvmfs/ directory
+  ${CVMFS_SOURCE_DIR}/auto_umount.cc
+  ${CVMFS_SOURCE_DIR}/backoff.cc
+  ${CVMFS_SOURCE_DIR}/cache.cc
+  ${CVMFS_SOURCE_DIR}/catalog.cc
+  ${CVMFS_SOURCE_DIR}/catalog_counters.cc
+  ${CVMFS_SOURCE_DIR}/catalog_mgr.cc
+  ${CVMFS_SOURCE_DIR}/catalog_mgr_ro.cc
+  ${CVMFS_SOURCE_DIR}/catalog_mgr_rw.cc
+  ${CVMFS_SOURCE_DIR}/catalog_rw.cc
+  ${CVMFS_SOURCE_DIR}/catalog_sql.cc
+  ${CVMFS_SOURCE_DIR}/compat.cc
+  ${CVMFS_SOURCE_DIR}/compression.cc
+  #${CVMFS_SOURCE_DIR}/cvmfs.cc
```

Comment:
I have maintained the commented lines in case in the future we can remove the comment, but is that a good idea?